### PR TITLE
Adding Armstrong Number and Automorphic Number

### LINKED ISSUE #1136: Armstrong Number and Automorphic Number in Kotlin
Please Assign it to me.

### PR DESCRIPTION
**Pull request by shubhammantri1**
Fixes #1136 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Adding Armstrong Number Algorithm in Kotlin
- Adding Automorphic Number Algorithm in Kotlin

#### Languages Used:
- Kotlin

#### Files Added:
- Armstrong_Number.kt
- Automorphic_Number.kt

### CODE REVIEW

File: Armstrong_Number/Armstrong_Number.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+    return count
+}
+
+//Function to check Armstrong Number
+fun armstrong(Num: Int, digits: Int):Int {
+    var sum = 0
+    var num = Num
+    while(num > 0) {
+        var digit =  num%10
```
Comment (MastersAbh) [inline Armstrong_Number/Armstrong_Number.kt]: space around %

Comment (MastersAbh) [reply Armstrong_Number/Armstrong_Number.kt]: only one space after =

File: Automorphic_Number/Automorphic_Number.kt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+fun main() {
+    //Let's take a number 36
+    var num = 76
+    var sqre = num*num
```
Comment (MastersAbh) [inline Automorphic_Number/Automorphic_Number.kt]: space around *

**Comment by jainaman224** [comment]
Hey @shubhammantri1,
What happened?

**Pull request closed without merging**